### PR TITLE
improve aesthetic documentation

### LINKED ISSUE #3293: Improve aesthetic documentation
All of the aesthetic help pages, including [colour, fill and alpha](https://ggplot2.tidyverse.org/reference/aes_colour_fill_alpha.html), [grouping](https://ggplot2.tidyverse.org/reference/aes_group_order.html), [linetype, size, and shape](https://ggplot2.tidyverse.org/reference/aes_linetype_size_shape.html), and [position aesthetics](https://ggplot2.tidyverse.org/reference/aes_position.html), have minimal text (they are almost all examples). I think there is room for an explanation of these aesthetics, much of which is currently in the form of comments between examples. This is a documentation issue intended for the tidyverse dev day (#3267). 

In particular, I think they each could use:

- A general explanation of what they change about common geometries (col usually changes the line colour, fill the inside colour, etc.).
- Links to the scales that map them.
- Links to the geometries where they are commonly used.

### PR DESCRIPTION
**Pull request by rivaquiroga**
Closes #3293

### CODE REVIEW

File: R/aes-colour-fill-alpha.r [hunk 1]
```diff
@@ -1,10 +1,23 @@
...
+#' of aesthetics: `colour`, `fill` and `alpha`.
+#'
+#' @details
+#' The `colour` aesthetic changes the colour of a line (or outline),
+#' `fill` changes the interior colouring of a geom, `alpha`
+#' changes its transparency.
+#'
+#' `colour` and `fill` aesthetics can be set with a color name or
+#' an rgb specification. The alpha aesthetic value goes from 0 to 1,
+#' where 0 is completely transparent and 1 is not transparent at all.
```
Comment (clauswilke) [inline R/aes-colour-fill-alpha.r]: Instead of writing "is not transparent at all", write "is completely opaque".

**Comment by clauswilke** [comment]
This PR and #3786 by @kelseymoty touch the same file and cannot be both merged without conflict. Since #3786 is smaller in scope, it should probably be merged first.

**Comment by clauswilke** [comment]
@rivaquiroga PR #3786 has now been merged. Do you want to update your PR so it can be merged as well? The first step would be to merge in the current master branch and fix any conflicts. You may simply want to discard your changes to the file `aes-colour-fill-alpha.r`, or maybe carry over only the changes to examples.

**Comment by thomasp85** [comment]
@clauswilke do you want to finish this off?

**Comment by clauswilke** [comment]
Sure, I'll do it.

**Comment by clauswilke** [comment]
@paleolimbot Could you look this over, since you filed the original issue? I'm done with cleaning this up.

File: R/aes-linetype-size-shape.r [hunk 2]
```diff
@@ -1,64 +1,74 @@
...
-#' This page demonstrates the usage of a sub-group
-#' of aesthetics; linetype, size and shape.
+#' @description
+#' The `linetype`, `size`, and `shape` aesthetics modify the appearance of
+#' lines and/or points. They also apply to the outlines of polygons (`linetype`
+#' and `size`) or to text (`size`).
 #'
+#' The `linetype` aesthetic can be specified with either an integer (0-6), a
+#' name (0 = blank, 1 = solid, 2 = dashed, 3 = dotted, 4 = dotdash, 5 = longdash,
+#' 6 = twodash), a variable source (for grouping), or a string of an even number
```
[the remark this answers is not in the document]
Comment (clauswilke) [reply R/aes-linetype-size-shape.r:10]: Yeah, I didn't write that phrase, and it confused me also. I think it means we can map a variable onto the aesthetic. I think I can rephrase.

**Comment by clauswilke** [comment]
Thanks @rivaquiroga, this will be part of the next ggplot2 release.

**Pull request merged**